spec/classes.md

```diff
@@ -1723,6 +1723,7 @@ method_modifier
     | 'override'
     | 'abstract'
     | 'extern'
+    | 'async'
```

Comment:
@gafter I see that the code treats `async` as a modifier. Can you confirm that's the case from the language perspective as well? Thanks

Reply:
Yes, the language spec treats `async` as a modifier that can be placed anywhere among the other modifiers.